backend/src/apiserver/data/context/ranking.py

```diff
@@ -136,6 +139,50 @@ async def add_new_training(dsrc: Source, new_event: NewTrainingEvent) -> None:
         )
 
 
+async def add_new_training(dsrc: Source, new_event: NewTrainingEvent) -> None:
```

Comment:
Deze functie heeft dezelfde naam als functie hierboven, is dat de bedoeling?

Reply:
oeps! dit waren twee dezelfde functies, ik heb er 1 van verwijderd in de nieuwste commit